src/Plugin.js

```diff
@@ -128,6 +128,31 @@ class Plugin {
     getFixers() {
         return [];
     }
+
+    /**
+     * For a "transformer" type of plugin, this returns a list of Transformer
+     * classes that this plugin implements. Note this is the class, not an
+     * instance of the class. The linter may need to instantiate this transformer
+     * multiple times.
+     *
+     * @returns {Array.<Class>} list of Transformer classes implemented
```

Comment:
`Array.<Transformer>` is an array of Transformer instances. In this case, we want an array of Class that implements Transformer. Something like `Array.<Class extends Transformer>` but jsdoc doesn't like that.

Reply:
Okay, so I confused "implements" with "extends". The point still stands. The return value is an array of classes, not instances. Using ts notation, it would be something like `Array<new () => Transformer>`, but jsdoc doesn't support that syntax either so all we can do is say `Array.<Class>`.

Reply:
Oh right, I got this part wrong. Apologies.